webmachine.gemspec

```diff
@@ -17,7 +17,9 @@ Gem::Specification.new do |gem|
 
   gem.add_runtime_dependency(%q<i18n>, [">= 0.4.0"])
   gem.add_runtime_dependency(%q<multi_json>)
-  gem.add_runtime_dependency(%q<as-notifications>, ["~> 1.0"])
+  gem.add_runtime_dependency(%q<as-notifications>, ["~> 1.0.2"])
```

Comment:
This should actually be `["~> 1.0", ">= 1.0.2"]`, as if you just say `"~> 1.0.2"` it will not allow anything from the 1.1 series.

Reply:
Thanks, good spot. I had to do some reading to understand the pair of `["~> 1.0", ">= 1.0.2"]`. Is that effectively equivalent to `[">= 1.0.2", "< 2.0"]`? If so it might be clearer to state that.

Reply:
(And yes, aside of this I think it's good to go)

Reply:
Yes, you're right @rgarner, `[">= 1.0.2", "< 2.0"]` is much clearer.